admin/sql/create_tables.sql

```diff
@@ -101,6 +101,14 @@ CREATE TABLE user_timeline_event (
   created               TIMESTAMP WITH TIME ZONE DEFAULT NOW() NOT NULL
 );
 
+CREATE TABLE hide_user_timeline_event (
+    id           SERIAL,  --PK
+    user_id      INTEGER, --FK to "user"
+    event_type   hide_user_timeline_event_type_enum,
+    event_id     INTEGER,
```

Comment:
I think it makes sense to add a comment here that this event_id is a row id. It can be a row id of a pin in the pins table or a timeline event in that table etc.